### ISSUE TITLE
Space between axis line and the first bar

### ISSUE THREAD

**Issue by RavalikaP** [apache/echarts#16489]
### Version

5.2.2

### Link to Minimal Reproduction

https://echarts.apache.org/examples/en/editor.html?c=bar-background

### Steps to Reproduce

1. Create any basic bar chart
2. Observed that there is space between axis line and the bar

### Current Behavior

Observed that there is space between axis line and the bar

### Expected Behavior

Need any setting to remove that spacing can be removed 
![image](https://user-images.githubusercontent.com/15928301/153421892-b1ebbfc6-0b22-4069-8fef-4d2abf348ea0.png)

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by Ovilia** [comment]
This is not the space between the bar and the axis line. You can change it by setting [barWidth](https://echarts.apache.org/en/option.html#series-bar.barWidth) to be `'100%'`.

**Comment by RavalikaP** [comment]
@Ovilia, I have given barWidth as 100% but we are not getting gap between each bar.
![image](https://user-images.githubusercontent.com/15928301/153535792-4308df93-1dd6-4fdb-92cd-acf899238606.png)

**Comment by Ovilia** [comment]
Yes, this is expected. It's not possible to remove the gap only for the first bar.

**Comment by helgasoft** [comment]
we can hack it, but why? Does it look prettier ? [Demo Code](https://echarts.apache.org/examples/en/editor.html?c=bar-background&code=PYBwLglsB2AEC8sDeAoWsAeBBDEDOAXMmurGAJ4gCmRA5AMYCGYVA5sAE7m0A0J6AE2aMiAbVoBZGL1i0AKgFcqM2gHUqAlXIAWClQDEOEFQGVmphdFoBdEgF8-6cjnxEksByTxUjVQrFF-YlJSITARAIBGACYABh5YOPjYSIBWZIAOZIB2ZMjIvIBmWOtHELJKGlkAI0YOGVqOVQgBMG06ADZYgFIGuoBxRhACeKDPENRy2DCI0Xj50qD0Cmo6Rr6mlra6ABYe3jGSWzsAbiA)
![image](https://github.com/apache/echarts/assets/13038071/de96433e-f259-4c95-b6e0-10048fb296d4)
